locale/en/blog/announcements/interactive-2016-north-america-schedule.md

```diff
@@ -7,13 +7,13 @@ slug: interactive-2016-north-america-schedule
 layout: blog-post.hbs
 ---
 
-> IBM, Netflix, Microsoft and leading community experts to showcase the current and future of Node.js
+> IBM, Netflix, Microsoft, and leading community experts to showcase the current and future of Node.js
```

Comment:
ah the dreaded oxford comma
while it's not technically wrong, it's debatable whether we should add it